components/brave_rewards/resources/ui/reducers/publishers_reducer.ts

```diff
@@ -19,16 +19,12 @@ const publishersReducer: Reducer<Rewards.State | undefined> = (state: Rewards.St
 
       state.autoContributeList = action.payload.list
       break
-    case types.ON_EXCLUDED_PUBLISHERS_NUMBER: {
+    case types.ON_EXCLUDED_LIST: {
       state = { ...state }
-      let num = parseInt(action.payload.num, 10)
-
-      if (isNaN(num)) {
-        num = 0
+      if (action.payload.list) {
+        state.excludedList = action.payload.list
+        chrome.send('brave_rewards.getContributionList')
```

Comment:
why do we need to call contribution list here? If I am tracking it correctly `ON_EXCLUDED_LIST` is called when you call getter for excluded list

Reply:
oh I see. It's like this because `OnExcludedSitesChanged` goes through the same flow. Exclude should have callback that would tell UI to fetch new excluded list and AC list. This way we don't get AC list unnecessary

Reply:
done